devices/HapticGlove/include/SenseGloveHelper.hpp

```diff
@@ -0,0 +1,281 @@
+/**
+ * @file SenseGloveHelper.hpp
+ * @authors Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#ifndef SENSE_GLOVE_HELPER_HPP
+#define SENSE_GLOVE_HELPER_HPP
+
+#include <Eigen/Dense>
+
+// std
+#include <array>
+#include <iostream>
+#include <memory>
+
+// YARP
+#include <yarp/os/LogStream.h>
+#include <yarp/sig/Vector.h>
+
+// Sense Glove
+#include <DeviceList.h>
+#include <SenseGlove.h>
+
+/**
+ * GloveControlHelper is an helper class for controlling the glove.
+ */
+namespace senseGlove {
+class SenseGloveHelper;
+const std::string LogPrefix = "senseGlove::SenseGloveHelper::";
+const size_t PoseSize = 7;
+} // namespace senseGlove
+
+class senseGlove::SenseGloveHelper {
+
+  int m_forceFbDof; /**< Number of the actuated motors Dofs to produce force
```

Comment:
May be using [variable comment](https://google.github.io/styleguide/cppguide.html#Variable_Comments) style will be cleaner

Reply:
I added this style for the comments, if we decide to make [doxygen](https://www.doxygen.nl/manual/docblocks.html) we have the comments for the members and methods.

Reply:
If lines get too long after you apply clang format, you can move the comment before the definition (and adapt properly the special comment syntax)

Reply:
As you suggested, I added the comments before the member variables.

Reply:
I updated this, let me know if this is OK.